configs/projects/_shared-pe-bolt-server.rb

```diff
@@ -42,6 +42,7 @@
 # TODO: Can runtime projects use these updated versions?
 proj.setting(:rubygem_gettext_version, '3.2.9')
 proj.setting(:rubygem_deep_merge_version, '1.2.1')
+proj.setting(:rubygem_net_ssh_version, '5.2.0')
```

Comment:
I think it is ok to ship net-ssh in all branches.

Reply:
Should be